tests/kas_default_scenario.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { test, expect } from '@playwright/test';
+import login from '@lib/auth';
+import { config } from '@lib/config';
+import {
+  navigateToKafkaList,
+  navigateToConsumerGroups,
+  deleteKafkaInstance,
+  createKafkaInstance,
+  waitForKafkaReady,
+  navigateToAccess
+} from '@lib/kafka';
+
+const testInstanceName = 'test-instance-default';
+
+test.beforeEach(async ({ page }) => {
+  await login(page);
+
+  await navigateToKafkaList(page);
+
+  await expect(page.getByRole('button', { name: 'Create Kafka instance' })).toBeVisible();
+
+  if ((await page.getByText(testInstanceName).count()) > 0 && (await page.locator('tr').count()) === 2) {
+    // Test instance present, nothing to do!
+  } else {
+    await page.waitForSelector('[role=progressbar]', {
+      state: 'detached',
+      timeout: config.kafkaInstanceCreationTimeout
+    });
+
+    for (const el of await page.locator(`tr >> a`).elementHandles()) {
+      const name = await el.textContent();
+
+      if (name !== testInstanceName) {
+        await deleteKafkaInstance(page, name);
+      }
+    }
+
+    if ((await page.getByText(testInstanceName).count()) === 0) {
+      await createKafkaInstance(page, testInstanceName);
+    }
+  }
+});
+
+test.afterAll(async ({ page }) => {
+  await navigateToKafkaList(page);
+
+  try {
+    await deleteKafkaInstance(page, testInstanceName);
+  } catch (error) {
+    //Ignore exception
+  }
+});
+
+// test_4kafka.py test_kafka_consumer_groups_empty
+test('test kafka consumer groups empty', async ({ page }) => {
+  await waitForKafkaReady(page, testInstanceName);
```

Comment:
Why do we need a new test file for these two tests instead of put them into `kas_with_instance.spec.ts` ?

Reply:
I too don't get why this test is not in that file and you needed a new one. 🤔

Reply:
This is because the beforeEach() method needed to be different in order to get to the necessary scenario for that test. It is not possible to skip certain parts of the beforeEach and therefore this is the approach

Reply:
What are the steps that need to be different here?

Reply:
Can I resolve this conversation @Frawless ?

Reply:
@asanzgom no, there is still open question regarding this

Reply:
This test was originally placed on `kas_messaging.spec.ts` file where the beforeEach method creates topic, Service Account and produces messages. We don't need that for these tests as they are based on a default, clean environment without Consumer Groups nor permissions for Producing/Consuming messages.

Reply:
Ok, but @Frawless was asking about `kas_with_instance.spec.ts` and not `kas_messaging.spec.ts`.

Reply:
these tests are about consumer group and ACL so probably messaging was the preferred file for them

Reply:
Can we resolve this conversation?

Reply:
Ok, moved both tests into `kas_with_instance.spec.ts`

Reply:
Can we resolve this conversation @Frawless @mmusil ?